pages/linux/compress.md

```diff
@@ -0,0 +1,32 @@
+# compress
+
+> Compress files using the Unix `compress` command.
+> More information: <https://www.unix.com/man-page/osx/1/compress/>.
```

Comment:
```suggestion
> More information: <https://manned.org/compress.1>.
```

Your link didn't work. Further, as it contains `osx`, I think it is for the OS X version, not the linux version.